views/api/index.pug

```diff
@@ -125,3 +125,9 @@ block content
6baZ0Lj6FWl9XVeKWXyhVfRL+90x347ym4tpItX4te8fb/hO0eba3LPZclKhCLbjGMW+bxFJt977znJtaekytK1iOsQ5JxTTi1lNNNdU0+qMRO3VmY3ebdq9Kdpe1rfsjNun4wl60fg8ew7TR5vi4a2UWanJeYagktIZAAAAAAAAAAAAAAGG+RiSHpvQbJW9rRopY4KcIvzUVn4nDZ7zfJafV0GKvLSIbA1tioN9eqOVelaJ8oR9NNfilmMfclL3nQ8Fw7VtkVeuvvaKq0L1XgAAAAAAPulUlGSnF4cWpRfc08p+9Hm1YtExLMTMTvD0vs/qCubSjcL79OMn545r35OIz45x5LV9F/itz0izYGpsU5vss1G6oVkv6SnOL86cl+k0dFwS8zS1fT/ANqrX12tEq4LtADIAAAAAAAAAAAAAA57Ckp1qcH0lUpxflKSX6mvLbalnqsby9SHCuiYYHn3eZVctWuM9jpRXkqUP9zruGV201f++ak1U/iyi5YIwAAAAAAABfO6aq5aTST+7OvFeXpJP9TkeKxtqbfx9l1o/wAuEyK9KVnvxpr6PbT7VVnH2Shl/wCVF3wS22S0fRA18dIVAdFsqgyAAAAAAAAAAAAADA57GooVqc30jOnJ/wCGSf6HjLG9Zh6rO07vUZwroRhlQO9G3cNWrt/fVKovJ04r5xkdbwu2+mj6KTVxtllEyxRgAAAAAABAX7ust3DSaOfvOtU9kqksfDByHE7c2osu9JG2KEuICSrfff8AslD+O/5ci44N+bb2QNd4YU6dKqmDIAAAAAAAAAAAAAAPoJZh6a0G9Ve1o1k88dOEn5tc/jk4bPTkyWr6L/FbmpEu+avq2Sq3fXpLxRvYrp9RV9uZQb8M8S9qL3g2baZx/wAq7XY+kWVQdCrAAAAAAAHPY2s61WFGmsynKNOPnJ4NeXJ8Ok39HqteaYh6a0uzjQoU6EOkIRgv8KwcRlvN7zafNf0ry1iHaPD2rfff+yW/8d/y5FzwX82fZA1/hhTh0irDLAAAAAAAAAAAAAAABdG5vWPS2krWT9ajJuPf6OfNe6XEvccvxfDyZeePP7rfRZN68s+SwipTXS1fTqd1Qnb1VmM4uL713NeKfM94stsdovHk8ZKc9dpedNf0avZXEreuua5xl92cX0lHwfwOz02emopzVUeXHNJ2lrTe1hlgAAAAEy3TTprVIKcU24VFTb7JYzleOOIrOLRP+P09UrRz+Ivg5NdMmRW++/8AZLf+O/5ci54L+bPsga/wwpw6RVhlgAAAAAAAAAAAAAAA3Wx+uysLyFwsuP2KsV96Euvu5NeRE1unjNhmPPybsOT4dol6LtLiFWEatOSnCUVOEk8pprKaZxtqzW01nyXsWi0bw5Tyy0O1uy9vqNH0dX1ZrnSqJZnB/rF9qJWl1V9Pbmr2aM2GuSPqo7aTZe8sJ8Nem3Dnw1YpulJefY/BnU6bW489d4nr6KjJgvTu0hMaQwBnYfVOnKTUYxcm+iSbk/JLqebWiveWYjdKv/zvV/QKsqHXn6PiSrJd7i/lnJA/8pp+bl3/APiR/iZNt9mt0f09jf0KlalUouFanxKcJQfDxJSxnr6rkbc9sebDatZiekvOOLUyRu9HpnG+68jrG76DKt99/wCyW/8AHf8ALkXPBfzZ9kDX+GFOHSKsMsAAAAAAAAAAAAAAAGTHsz9Vhbsttfo0lZ3UvqpNKjJ9Kcm/st/uN+5+HSl4noOePi07+adpdRyzyyuaLzzRzey06Mjp2ZfFalCcXGcVKL5NNZT80zMTNesMTG/dENV3aaVWblCnK3k/+3NqP5HlL2YLDFxTUUjaZ3hFvo8du3Roqm5+jn1byaXjTi380S441f8AbDV/gR6uzabo7KLzVuKtTvS4YL5ZPF+M5Z8MRDMaGvnKX6JszYWa/wCmt4wfbJ5nUfnOWWV2bVZs0/PKTTDSnhhtsEf2bnFc2lKrHhqwjUXdKKkvieq3tWd4nZ5msT3hyxiksLklyRjv1ZfQZVvvv/ZLf+O/5ci54L+bPsga/wAMKcOkVYZYAAAAAAAAAAAAAAAAAwzCfbDbwqloo295mpQXKEknKrT8PxR8Oq7O4p9dwyMvz4+k/dM0+rmvy27LksL6jXpqrQqRqwfSUXlf7HOXx2pba0dVrW0WjpLsI8vTIAAAA+TA4aF5RnKUKdSM5QaVRKSbi30TS6HuaWrG8w8xaJnpLnPHR6ZMit99/wCyW/8AHf8ALkXPBfzZ9kDX+GFOHSKsMsAAAAAAAAAAAAAAAAAY2gDI2Gj63d2c+O1rSpPtSeYS8JRfJkfNpseaNrw2Uy2pO9ZWHo291rEby2z0zOlL4uEv0ZT5uDeeOU6mv/dCW2e8TR6i/aPRvunCUP0wV+Th2op+lJrq8U+bYR2v0l/2+29txBfNmmdJn/ZP9NnxqesOKvttpEOt9Rf92fpP8uTMaPPP6ZYnUY47y0Op71tOp8qEKlw+zC9HD80v9CZj4Rmnx9Gi2upHZAtod42o3acISVrB9Y05Pjfg6nX3YLbT8Lw4p3t1lCy6u9+3RoND1u5s6yr283GX3k23Ga6tTXaS8+mpmpy2j2aceW1J3hbegb0rGtFRuk7Wfb1nSflJLK8mvec9n4Tmp4OsLPHra28XRJI7X6S1n6fbe2vBP3NkP/Dz/sn+kj41PWED3va1ZXNtRjbXNGu1WcpKFWNRpcEllpPki04ThyY8szasx0Q9betqxESqs6Gekq0DAAAAAAAAAAAAAAAAAAAAAdWWTHswGRgbjIZYDDJhkDDBncAyD6sAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB//2Q==', height=40)
```

Comment:
Instead of the binary image, can you use a link to the image similar to the other examples on the page?